### PR TITLE
Handle AppVeyor renaming repository

### PR DESCRIPTION
**Pull request by zakame**
I just found out that AppVeyor does a `$repository_name ~= s/\./-/g` so
take account for that when regenerating the badge.

### CODE REVIEW

File: lib/Minilla/Project.pm [hunk 1]
```diff
@@ -650,6 +650,7 @@ sub regenerate_readme_md {
                     $image_uri->query_form( map { ( $_, $image_uri_qs{$_} ) } sort keys %image_uri_qs );
                     push @badges, "[![Build Status]($image_uri)]($build_uri)";
                 } elsif ($service_name eq 'appveyor') {
+                    $repository_name =~ s/\./-/g;
```
Comment (skaji) [inline lib/Minilla/Project.pm]: `$repository_name` is shared between for-loop. So you'd better copy it first.

Comment (zakame) [inline lib/Minilla/Project.pm]: Thanks, will add a test for that too.

**Comment by skaji** [comment]
LGTM

**Comment by zakame** [comment]
Thanks @tokuhirom @skaji ! 👍

**Comment by skaji** [comment]
@zakame Released https://metacpan.org/release/SKAJI/Minilla-v3.0.13

**Pull request merged**